tests/smoke-tests/test-block-finalized.ts

```diff
@@ -0,0 +1,90 @@
+import "@moonbeam-network/api-augment";
+import { expect } from "chai";
+import { describeSmokeSuite } from "../util/setup-smoke-tests";
+
+const debug = require("debug")("smoke:block-finalized");
+const wssUrl = process.env.WSS_URL || null;
+const relayWssUrl = process.env.RELAY_WSS_URL || null;
+const ethRpcUrl = process.env.ETH_URL || null;
+
+describeSmokeSuite(
+  `Parachain blocks should be finalized..`,
+  { wssUrl, relayWssUrl, ethRpcUrl },
+  (context) => {
+    it("should have a recently finalized block", async function () {
+      const head = await context.polkadotApi.rpc.chain.getFinalizedHead();
+      const block = (await context.polkadotApi.rpc.chain.getBlock(head)).toHuman() as any;
+      const diff = Date.now() - getBlockTime(block);
+      debug(`Last finalized block was ${diff / 1000} seconds ago`);
+      expect(diff).to.be.lessThanOrEqual(10 * 60 * 1000); // 10 minutes in milliseconds
+    });
+
+    // When 0.9.29 is live we can enable this test
+    // TODO: update the CI to provide ETH_URL env var
+    it.skip("should have a recently finalized eth block", async function () {
```

Comment:
This should not be skipped.
The Smoke tests need to be retro-compatible (see other tests like balance) to support any version

Reply:
Done